app/controllers/repair_requests_controller.rb

```diff
@@ -89,6 +94,12 @@ def set_property
     @property = Hackney::Property.find(params[:property_ref])
   end
 
+  def get_cautionary_contact_alert_codes(cautionary_contacts)
+    cautionary_contacts.map do |cautionary_contact|
+      cautionary_contact.alert_code
+    end.join("; ")
+  end
```

Comment:
It's fine the way it is, but just a tip: `cautionary_contacts.map(&:alert_code).join("; ")`

Reply:
Actually it would be better to have this inline, as it's used only once, on line 29.